src/handlers/applications.js

```diff
@@ -1,14 +1,86 @@
-const applicationService = require('../services/applications')
+const service = require('../services/applications')
+const schemas = require('../schemas/applications')
+const response = require('../utils/response')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/lucianopf/serverless-webhooks-service/pullRequest?prid=9084646)
```suggestion
const response = require("../utils/response")
```